railgun/controllers/configuration/ingress_utils.go

```diff
@@ -18,7 +18,7 @@ const KongIngressFinalizer = "configuration.konghq.com/ingress"
 // provided controller manager.
 func SetupIngressControllers(mgr ctrl.Manager) error {
 	netV1Ing := new(netv1.Ingress)
-	apiAvailable, err := isAPIAvailable(mgr, netV1Ing)
+	apiAvailable, err := IsAPIAvailable(mgr, netV1Ing)
```

Comment:
note: I made this public so that `main.go` could use it to short circuit the `UDPIngress` API on clusters which did not support it.